client/src/services/LoginService.js

```diff
@@ -1,50 +1,49 @@
+import ErrorMessage from 'models/ErrorMessages';
+
 export default class LoginService {
   static login = (data) => {
+
     const formData = new FormData();
 
     formData.append('username', data.username);
     formData.append('password', data.password);
 
-
-    // Configure POST request options
-    const fetchOptions = {
+    const postRequestOptions = {
       method: 'POST',
       body: formData,
     };
 
+
     return new Promise(async (resolve, reject) => {
       try {
-        // Sends the POST Request
-        // Catch network errors and thow them to the catch block
+
+        /* Sends the POST Request */
         const response = await fetch(
-          'http://localhost:80/api/login',
-          fetchOptions
-        ).catch(() => {
-          throw new Error('Encountered a network error. Try again soon.');
+          `${process.env.SERVER_URI}/api/login`,
+          postRequestOptions
+        ).catch(() => { // Fails to connect to API
+          throw new Error(ErrorMessage.NetworkError);
         });
 
-        // Convert Response body to JSON
-        const result = await response.json();
+        const responseJson = await response.json();
```

Comment:
'result' tells us its the by-product of an action/modification but doesn't tell us what it is

We can rename it to responseJson so that we can better tell this is the response in json format.